composer.json

```diff
@@ -34,7 +34,8 @@
         "sonata-project/easy-extends-bundle": "^2.5",
         "sonata-project/notification-bundle": "^3.3",
         "sonata-project/seo-bundle": "^2.5",
-        "symfony-cmf/routing-bundle": "^1.1 || ^2.0",
+        "symfony-cmf/routing": "^2.1-rc1",
```

Comment:
maybe add a pipe for older version as we still support 5.6

Reply:
I left only 2.1-rc1 here, because a main goal of PR is to show, that new version of routing-bundle works well with SonataPageBundle. I hope, that it will help to release routing-bundle faster ) Also, composer.json should not be updated after their release - deps are fine